src/components/Board/Board.component.js

```diff
@@ -104,7 +107,70 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
+      runLiveUnlock: false,
+      steps: [
+        {
+          target: 'body',
+          placement: 'center',
+          hideCloseButton: 'true',
```

Comment:
You should use a boolean here, not a string. It works currently because a non-empty string behaves exactly like `true` (it's "truthy") but if you put `'false'` the button would probably still be hidden.

```javascript
// The button is hidden.
hideCloseButton: 'true'
hideCloseButton: 'false'

// The button is visible.
hideCloseButton: false
```

(Sorry if you already know about truthy and falsy values.)

Reply:
No we did not know about truthy and falsy values! Thank you so much, it was very insightful! I will update it!